src/envoy/mixer/mixer_control.h

```diff
@@ -32,7 +34,8 @@ class HttpMixerControl final : public ThreadLocal::ThreadLocalObject {
   // The constructor.
   HttpMixerControl(const HttpMixerConfig& mixer_config,
                    Upstream::ClusterManager& cm, Event::Dispatcher& dispatcher,
-                   Runtime::RandomGenerator& random);
+                   Runtime::RandomGenerator& random,
+                   const std::string& stats_prefix, Stats::Scope& scope);
```

Comment:
Not need to pass in stats_prefix.    It is already in HttpMixerControl constructor.